superset-frontend/src/explore/components/PropertiesModal.tsx

```diff
@@ -159,7 +159,7 @@ function PropertiesModal({ slice, onHide, onSave }: InternalProps) {
 
   return (
     <form onSubmit={onSubmit}>
-      <Modal.Header closeButton>
+      <Modal.Header data-test="edit-modal" closeButton>
```

Comment:
Could be more specific (i.e. 'edit-properties-modal') if you can expect there to be more than one instance of modal in the code with `data-test="edit-modal"` - it just makes it easier to find the instance in the codebase.